### ISSUE TITLE
Missing Purpose Hexagon: Advertising

### ISSUE THREAD

**Issue by sevaan** [sidewalklabs/dtpr#10]
Data being collected to aid in advertising is a very real use that is not represented here.

**Comment by patrickkeenan** [comment]
We actually don't have any use cases that warrant that, if you can think of some example that would be something you could suggest being added. The key would be somewhere in cities that's happening now, so we could add it to the system.

**Comment by sevaan** [comment]
One use case is facial recognition for advertising. There are many examples of this happening across the world.

Some quickly Googled examples:

- https://www.telegraph.co.uk/technology/2018/11/08/facial-recognition-billboard-maker-bidooh-wins-contract-roll/
- https://www.iflscience.com/technology/now-advertising-billboards-can-read-your-emotions-and-s-just-start/

**Comment by jackielu** [comment]
I agree that there are use cases, just not ones that we have / would support at Sidewalk Labs. Sevaan - might you be interested in developing a concept for one and submitting a pull request?
